api-testing/multicloud-api/src/main/java/com/soda/utils/TextUtils.java

```diff
@@ -0,0 +1,30 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils;
+
+/**
+ *In This class text related common util
+ */
+public class TextUtils {
+    /**
+     * Returns true if the string is null or 0-length.
+     * @param str the string to be examined
+     * @return true if str is null or zero length
+     */
+    public static boolean isEmpty(CharSequence str) {
```

Comment:
Where are using/referencing this method call?
Can't we use the inbuilt library function?

Reply:
This is repeated questions. Already discuss with @kumarashit

Reply:
Same as above. Give proper reasoning for the questions asked.

Reply:
@himanshuvar This code use in test case method to check a string. This is a basic framework structure it will be use in next PR.